tools/README.md

```diff
@@ -0,0 +1,113 @@
+tools
+-----
+
+- `parallel_tests.sh`: Tool to execute the test suite in parallel, useful to
+  run multiple integration tests at the same time.
+- `startcluster.py`: This tool will generate a genesis file with a few
+  pre-funded accounts to be used as Raidne accounts, and then run a local
+  Geth private chain, useful for debugging.
+- `genesis_builder.py`: This tool will generate a genesis file.
```

Comment:
YOu link to this script here, but it's deleted.

Reply:
+1, nice catch